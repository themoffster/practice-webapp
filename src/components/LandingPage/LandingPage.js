import React from 'react'

const LandingPage = () => (
  <>
    <pre>foo</pre>
  </>
)

export default LandingPage
