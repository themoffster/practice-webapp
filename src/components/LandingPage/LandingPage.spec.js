import { render } from '@testing-library/react'
import LandingPage from './LandingPage'
import React from 'react'

const getWrapper = () => render(<LandingPage />)

describe('LandingPage', () => {
  it('matches the snapshot', () => {
    const { getFragment } = getWrapper()
    expect(getFragment).toMatchSnapshot()
  })
})
