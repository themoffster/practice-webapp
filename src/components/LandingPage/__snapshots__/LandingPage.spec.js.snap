// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LandingPage matches the snapshot 1`] = `undefined`;
